src/mach/load_command.rs

```diff
@@ -1014,18 +1056,18 @@ pub struct VersionMinCommand {
 }
 
 impl VersionMinCommand {
-    pub fn new(is_ios: bool) -> Self {
+    pub fn new(platform: Platform) -> Self {
         VersionMinCommand {
-            cmd: if is_ios {
-                LC_VERSION_MIN_IPHONEOS
-            } else {
-                LC_VERSION_MIN_MACOSX
-            },
+            cmd: platform as u32,
```

Comment:
Is this a new feature ? How is this as cast working since Platform isn’t repr(u32)